app/src/main/res/values/strings.xml

```diff
@@ -11,6 +11,8 @@
   <string name="patientsearch_go_back">Go back to home screen</string>
   <string name="patientsearch_name_or_phone">Patient\'s name or phone</string>
   <string name="patientsearch_age">Age: %s</string>
+  <string name="patientsearch_filter_by">Filter by</string>
+  <string name="patientsearch_label_age">AGE</string>
```

Comment:
Probably a small thing, but it is always recommended to capitalize strings in the UI using `textAllCaps` instead of normalizing them